.env

```diff
@@ -0,0 +1,2 @@
+BUTTONDOWN_API_URL=https://api.buttondown.email/v1/
+BUTTONDOWN_API_KEY=e1c25c86-0539-4fae-a806-bb593f372e9a
```

Comment:
sorry didn't mean to submit the PR to this repo. This key has been invalidated as well